### DISCUSSION TITLE
Launch scene-viewer on landscape position

### DISCUSSION THREAD

**Discussion by sercandemirtas** [google/model-viewer#2546] [General]
Hello all!

Is it possible to launch scene-viewer on landscape mode? 

I mean when we tap ar-button on model-viewer or call scene-viewer via intent scripts we need to put our device in a portrait position to scan the area.

I did research on the modelviewer.dev and ARCore developer page but I couldn't find anything.

Any suggestion or solution would be great! 

ps: we can launch webxr on both portrait and landscape.

**Comment by sercandemirtas** [comment]
In this videos Im using "intent://arvr.google.com/scene-viewer/1.0" to call scene-viewer.

Samsung Galaxy Tab S7 / current version: T870XXU2BUE2/T870OXM2BUE2

This is what it need to be.

https://user-images.githubusercontent.com/68244546/124489498-bdec4b80-ddb9-11eb-8e66-cc32e75b5955.mp4



But sometimes it opens like this


https://user-images.githubusercontent.com/68244546/124489638-e3795500-ddb9-11eb-8068-16711cbf1124.mp4

**Reply by elalish** [reply]
@tpsiaki Does Scene Viewer support landscape mode?

**Reply by sercandemirtas** [reply]
is there any news about it ?

**Reply by sercandemirtas** [reply]
@tpsiaki @elalish Hello again friends! 
I wanted to remind you of this issue.

**Comment by sercandemirtas** [comment]
Any update on this topic ?

**Reply by elalish** [reply]
I wouldn't hold your breath for this feature. Does WebXR suit your needs? It's available everywhere Scene Viewer is, and quite a lot more devices besides.

**Reply by sercandemirtas** [reply]
Thanks for to inform. Im using intent functions to open AR for my 3D model. That's why I choose scene-viewer. It is possible to call WebXR with intent functions?

**Reply by elalish** [reply]
No, WebXR is better because there is no intenting out to a separate app; you stay in the browser. That also means you can retain all your state, analytics, HTML UI, etc. Of course  if you don't want to write low-level rendering  code, you might prefer to use WebXR through model-viewer.
